modules/cnf-image-based-upgrade-troubleshooting.adoc

```diff
@@ -0,0 +1,289 @@
+  Normal   Scheduled         56s                default-scheduler  Successfully assigned default/db-1234 to sno1.example.lab
+  Warning  FailedMount       24s (x7 over 55s)  kubelet            MountVolume.SetUp failed for volume "pvc-1234" : rpc error: code = Unknown desc = VolumeID is not found
+----
+--
+
+Resolution::
+You must include `logicalvolumes.topolvm.io` in the application `Backup` CR.
+Without this resource, the application restores its persistent volume claims and persistent volume manifests correctly, however, the `logicalvolume` associated with this persistent volume is not restored properly after pivot.
++
+.Example Backup CR
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: small-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  includedClusterScopedResources:
+  - persistentVolumes <1>
+  - volumesnapshotcontents <1>
+  - logicalvolumes.topolvm.io <1>
+----
+<1> Required to restore the persistent volumes for your application. 
+
+[id="ztp-image-based-upgrade-troubleshooting-lvms-restore_{context}"]
+=== Missing {lvms}-related fields in Restore CR
+
+Issue::
+The expected resources for the applications are restored but the persistent volume contents are not preserved after upgrading.
++
+--
+.Example output before pivot
+[source,terminal]
+----
+$ oc get pv,pvc,logicalvolumes.topolvm.io -A
+----
+
+[source,terminal]
+----
+NAME                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM            STORAGECLASS   REASON   AGE
+persistentvolume/pvc-1234   1Gi        RWO            Retain           Bound    default/pvc-db   lvms-vg1                4h45m
+
+NAMESPACE   NAME                           STATUS   VOLUME     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+default     persistentvolumeclaim/pvc-db   Bound    pvc-1234   1Gi        RWO            lvms-vg1       4h45m
+
+NAMESPACE   NAME                                AGE
+            logicalvolume.topolvm.io/pvc-1234   4h45m
+----
+
+.Example output after pivot
+[source,terminal]
+----
+$ oc get pv,pvc,logicalvolumes.topolvm.io -A
+----
+
+[source,terminal]
+----
+NAME                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM            STORAGECLASS   REASON   AGE
+persistentvolume/pvc-1234   1Gi        RWO            Delete           Bound    default/pvc-db   lvms-vg1                19s
+
+NAMESPACE   NAME                           STATUS   VOLUME     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+default     persistentvolumeclaim/pvc-db   Bound    pvc-1234   1Gi        RWO            lvms-vg1       19s
+
+NAMESPACE   NAME                                AGE
+            logicalvolume.topolvm.io/pvc-1234   18s
+----
```

Comment:
Would be good to break the command out so that what the command does is described and so that the headings ("example output") are right above outputs instead of commands.